.dockerignore

```diff
@@ -74,6 +74,8 @@ cpu.out
 /VERSION
 /.air
 /.go-licenses
+Dockerfile
+Dockerfile.rootless
```

Comment:
Why no leading slash? Imho it's better to exactly ignore these files in the root, not ignore them anywhere in the tree.

Reply:
Actually I think maybe they don't need to be ignored? I guess there should be nothing wrong by keeping them

Reply:
IMO they should be ignored - if they aren't you end up rebuilding entire container because of a change in a comment in Dockerfile, which I just find annoying if nothing meaningful changes. When working on the dockerfile it's just annoying as let's say you changed package in the assembly stage or directives for binds/cache in `RUN make` step. Now the COPY digest differs so it ends up running again and has to build gitea from scratch despite nothing of relevance to the build being changed.
You don't hit this unless you're tinkering with the container builds and it was much worse pre-cache since building took 15-30 minutes on my machine due to having to download packages over and over.